### DISCUSSION TITLE
CommunityToolkit.WinUI.Animations namespace not found

### DISCUSSION THREAD

**Discussion by gmgallo** [CommunityToolkit/Windows#275] [Q&A] [answered]
Hi, I'm trying to learn how to apply Implicit animations on controls (other than the Canvas in the sample, if that is possible), but the namespace is not available as shown in the sample code. The only sub namespace I can see after WinUI is .UI. Where is it?
My environment is Windows 10, Microsoft Visual Studio Community 2022 (64-bit) - Preview Version 17.8.0 Preview 7.0

**Comment by michael-hawker** [comment]
@gmgallo if you're using WinUI 3 then you need the `CommunityToolkit.WinUI.Animations` package, for UWP it's the `CommunityToolkit.UWP.Animations`. It seems like you may be looking at the old packages/namespaces from 7.x?

**Comment by gmgallo** [comment]
Hi Michael, thanks for the prompt answer. My project is a MVVM application in WinUI 3 created with CommunityToolkit.Mvvm 8.2. I looked on github for the CommunityToolkit.WinUI.Animations package but it's not there. The only toolkits available in github are the ones already installed in my project dependies that you can see in this screenshot:
![image](https://github.com/CommunityToolkit/Windows/assets/11802515/30a36f66-625b-42f4-ac99-78cb0134dc93)
Is it available as a separate download?

**Reply by michael-hawker** [reply, marked as answer]
🦙 It's available via NuGet here: https://www.nuget.org/packages/CommunityToolkit.WinUI.Animations

**Reply by gmgallo** [reply]
It installed from the command line without a glitch. Thanks!
